fhir-persistence-jdbc/src/test/resources/config/date/extension-search-parameters.json

```diff
@@ -510,7 +510,7 @@
                     "Basic"
                 ],
                 "type": "date",
-                "expression": "Basic.extension.where(url='http://example.org/missing-Period').valuePeriod",
+                "expression": "Basic.extension.where(url='http://example.org/missing-Period').value",
```

Comment:
I double checked these files for consistency 
./fhir-server/liberty-config/config/default/extension-search-parameters.json
./fhir-server/liberty-config-tenants/config/tenant1/extension-search-parameters.json
./fhir-server/liberty-config-tenants/config/tenant2/extension-search-parameters.json

There may be others I missed, and causing the IT failures?